.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### What?
+
+<!-- What changes are made? -->
+<!-- (if there are many significant changes, a list might be a good format)  -->
+
+<br>
+
+### Why?
+
+<!-- Why are these changes made? -->
+
+<!-- If there is one, please link to the ticket where this issue is described. -->
+
+_Referenced ticket [here]()_
```

Comment:
Maybe we should put this in the PR title and or/branch name instead? So the gitub-jira integration can pick it up

Reply:
But if so i think we should put a reminder in the comments here!

Reply:
I think we could have both maybe? It's nice with a link I think, so it's easy to get to the ticket, but I agree that we should add it to the titles!